ui/apps/dashboard/src/app/(auth)/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -1,12 +1,41 @@
+import Link from 'next/link';
 import { SignIn } from '@clerk/nextjs';
 
 import SplitView from '@/app/(logged-out)/SplitView';
+import { Alert } from '@/components/Alert';
+
+type SignInPageProps = {
+  searchParams: { [key: string]: string | string[] | undefined };
+};
+
+export default function SignInPage({ searchParams }: SignInPageProps) {
+  const errorMessages = {
+    unauthenticated: 'Could not successfully signed you in. Please sign in and try again.',
```

Comment:
Maybe something like this?
```suggestion
    unauthenticated: 'Could not resume your session. Please sign in again.',
```